README.md

```diff
@@ -115,6 +116,7 @@ The following values can be configured:
 * `id`: The MMKV instance's ID. If you want to use multiple instances, use different IDs. For example, you can separate the global app's storage and a logged-in user's storage. (required if `path` or `encryptionKey` fields are specified, otherwise defaults to: `'mmkv.default'`)
 * `path`: The MMKV instance's root path. By default, MMKV stores file inside `$(Documents)/mmkv/`. You can customize MMKV's root directory on MMKV initialization (documentation: [iOS](https://github.com/Tencent/MMKV/wiki/iOS_advance#customize-location) / [Android](https://github.com/Tencent/MMKV/wiki/android_advance#customize-location))
 * `encryptionKey`: The MMKV instance's encryption/decryption key. By default, MMKV stores all key-values in plain text on file, relying on iOS's/Android's sandbox to make sure the file is encrypted. Should you worry about information leaking, you can choose to encrypt MMKV. (documentation: [iOS](https://github.com/Tencent/MMKV/wiki/iOS_advance#encryption) / [Android](https://github.com/Tencent/MMKV/wiki/android_advance#encryption))
+* `webFallbackStorage`: An alternative storage solution conforming to the `Storage` type. This is utilized as a fallback for web environments when, for instance, a user disables `localStorage` in their browser. This feature enables your application to manage such situations gracefully, offering eg. in-memory storage as an alternative (see example below). Please note that this property is optional; by default, an error will be thrown in case of a disabled localStorage.
```

Comment:
Do you really think we should provide a custom fallback here? I feel like the user should provide a fallback for the entire MMKV thing if he really wants to use a different lib. Maybe we can just add a prop like "`fallbackToInMemoryMap: boolean`" which just uses a local Map and doesnt write to storage? (With a proper disclaimer)

Reply:
Good point. I was also wondering if we need to ask the user to provide a fallback, or if it's too much.

But, what if we don't ask the user anything? It will be enabled by default. We can console.warn a message like DataDog does:

<img width="408" alt="image" src="https://user-images.githubusercontent.com/9088288/230643553-fc9a2979-1e16-4cab-afd4-95be68564d89.png">

Something like:
```
Local storage is disabled your experience will be limited to in-memory storage
```

If you think about it, who would want to get a crash (especially on production)? The library can provide be default a `Map`, and maybe for the convenience of the developer, return some flag to be consumed later to display some kind of message/popup like Google does, for example:

<img width="522" alt="image" src="https://user-images.githubusercontent.com/9088288/230643996-8d8d9a5d-8f86-4877-9b2e-94002375cd1b.png">

It's still better than a crash.

I was wondering about something like:

```typescript
return {
    clearAll: () => {},
    delete: (key) => {},
    set: (key, value) => {},
    //  (...) other functions,
    inMemoryFallback: boolean // for web depends on localStorage permission, for native always true
```

Let me know what do you think about it

Reply:
Yea I think this is better. I'm not sure if I want that flag though, this is a bit confusing for everyone that didn't think of this behaviour... Maybe we want web devs to intentionally check for localStorage?